### ISSUE TITLE
[Panel/UI] Accessibility tests

### ISSUE THREAD

**Issue by distantnative** [getkirby/kirby#76]
We should look into accessibility testing tools like Axe to check and improve the accessibility of our components.

@olach:
> Apart from regular accessibility, being able to navigate the ui with the keyboard would be a good enhancement.
>
> This is a good resource with short to the point examples:
https://www.w3.org/TR/wai-aria-practices/examples/
>
> More info:
> https://www.w3.org/TR/wai-aria-practices/

@bastianallgeier 
> I already added an issue to integrate accessibility tests: k-next/panel#56 because I really think that we should have a11y at the core of everything we build. @fabianmichael made sure that all our new colors meet the WCAG 2.0 AA guidelines for contrast and I started introducing basic tab navigation accessibility for every navigation element in the latest commit: af85e14
> 
> We are all no experts in this field, so any kind of help is very much appreciated!

**Comment by bastianallgeier** [comment]
Moved to: https://github.com/k-next/starterkit/issues/198
